workflows/bacterial_genomics/quality_and_contamination_control/quality_and_contamination_control.ga

```diff
@@ -0,0 +1,811 @@
+        },
+        {
+            "class": "Organization",
+            "name": "abromics-consortium",
+            "url": "https://www.abromics.fr/"
+        },
+        {
+            "alternateName": "pimarin",
+            "class": "Person",
+            "identifier": "https://orcid.org/0000-0002-8304-138X",
+            "name": "Pierre Marin"
+        },
+        {
+            "alternateName": "clsiguret",
+            "class": "Person",
+            "identifier": "https://orcid.org/0009-0005-6140-0379",
+            "name": "Clea Siguret"
+        }
+    ],
+    "format-version": "0.1",
+    "release": "1.0",
+    "license": "GPL-3.0-or-later",
+    "name": "quality_and_contamination_control",
+    "steps": {
+        "0": {
+            "annotation": "genomic,input,raw_reads,forward,fastq, paired R1 input reads",
+            "content_id": null,
+            "errors": null,
+            "id": 0,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "genomic,input,raw_reads,forward,fastq, paired R1 input reads",
+                    "name": "paired_r1"
+                }
+            ],
+            "label": "paired_r1",
+            "name": "Input dataset",
+            "outputs": [],
+            "position": {
+                "left": 15.05527979503197,
+                "top": 0
+            },
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"format\": [\"fastq\", \"fastq.gz\", \"fastqsanger\", \"fastqsanger.gz\"], \"tag\": \"\"}",
+            "tool_version": null,
+            "type": "data_input",
+            "uuid": "7322da44-fc1a-437f-ac1e-ac9096bdf779",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "1": {
+            "annotation": "genomic,input,raw_reads,reverse,fastq, paired R1 input reads",
+            "content_id": null,
+            "errors": null,
+            "id": 1,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "genomic,input,raw_reads,reverse,fastq, paired R1 input reads",
+                    "name": "paired_r2"
+                }
+            ],
+            "label": "paired_r2",
+            "name": "Input dataset",
+            "outputs": [],
+            "position": {
+                "left": 11.986858736752191,
+                "top": 108.04692569668924
+            },
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"format\": [\"fastq\", \"fastq.gz\", \"fastqsanger\", \"fastqsanger.gz\"], \"tag\": \"\"}",
+            "tool_version": null,
+            "type": "data_input",
+            "uuid": "0f8fa828-7694-4a4f-909c-3d2d468013af",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "2": {
+            "annotation": "Select the Kraken2 database used to the taxonomic assignation, the same database type need to be used by Bracken",
```

Comment:
Since these are the same values I would just use a single parameter. This way it is also impossible for users to select conflicting values.

Reply:
Thanks, i didn't think about it  :flushed: